packages/ui/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,207 @@
+                                    isTablet && `text-lg font-bold`,
+                                )}
+                                href="/about"
+                                locale={locale ?? false}
+                            >
+                                {/* TODO: Use translations */}
+                                About
+                            </Link>
+                        )}
+                        {hasStandaloneContactsPage && (
+                            <Link
+                                className={twMerge(
+                                    'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                    isTablet && `text-lg font-bold`,
+                                )}
+                                href="/contacts"
+                                locale={locale}
+                            >
+                                {/* TODO: Use translations */}
+                                Contacts
+                            </Link>
+                        )}
+                    </div>
+
+                    <div className="flex flex-col md:flex-row md:items-center gap-12 md:gap-4">
+                        {Boolean(isSearchEnabled && onSearch) && (
+                            <a className="hidden md:flex" href="#" onClick={handleSearch}>
+                                <MagnifyingGlassIcon className="w-[20px] h-[20px] text-gray-600 hover:text-gray-800" />
+                            </a>
+                        )}
+                        <div className="flex items-start md:items-center flex-row-reverse md:flex-row bg-gray-50 md:bg-transparent p-6 md:p-0 gap-4 justify-between md:justify-start">
+                            {languages.length > 0 && (
+                                <LanguagesDropdown
+                                    hasError={hasError}
+                                    languages={languages}
+                                    locale={locale}
+                                    currentCategory={currentCategory}
+                                    currentStory={currentStory}
+                                />
+                            )}
+                            {externalSiteLink && (
+                                <a
+                                    className="label-large text-gray-600 hover:text-gray-800 flex items-center shrink-0"
+                                    href={externalSiteLink}
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                    data-title={
+                                        <>
+                                            {extractDomainFromUrl(externalSiteLink)}
+                                            <ArrowUpRightIcon className="ml-1 w-2 h-2" />
+                                        </>
+                                    }
```

Comment:
What is this needed for? The text content of the link is already picked up by assistive technologies. Also, you can't put JSX into data-attributes anyway 🤔